toxcore/tox.h

```diff
@@ -2403,6 +2403,22 @@ typedef enum TOX_CONFERENCE_STATE_CHANGE {
      */
     TOX_CONFERENCE_STATE_CHANGE_PEER_NAME_CHANGE,
 
+    /**
```

Comment:
strange naming?

above in group.h it "chat" old style:
CHAT_CHANGE_PEER_NAME

in tox.h now its suddenly "conference":
TOX_CONFERENCE_STATE_CHANGE_PEER_NAME_CHANGE

also the enums are renamend and the values shuffled. why? (this is why enums suck, values get assigned in some magical way)

Reply:
I didn't change the name in this PR and I'm not going to change it now, because this whole event will go away later. The enum values are shuffled because this is an incompatible change anyway, and it's easier to remove the ones at the end than at the beginning (removing the beginning would be another ABI breaking change).